docs/commands/rpc-protocol.md

```diff
@@ -2840,30 +2839,61 @@ Clear work peers node list until restart
 ### work_validate 
 Check whether **work** is valid for block  
 
+!!! warning "Semantics change in V21.0"
+    In V21.0, when the optional **difficulty** is *not* given, **valid** is not included in the response.
+
+    Use the new response fields **"valid_all"** and **"valid_receive"** taking into account the subtype of the block using this work value.
+
 **Request:**
 ```json
 {
   "action": "work_validate",
   "work": "2bf29ef00786a6bc",
-  "hash": "718CC2121C3E641059BC1C2CFC45666C99E8AE922F7A807B7D07B62C995D79E2",
-  "difficulty": "ffffffd21c3933f3"
+  "hash": "718CC2121C3E641059BC1C2CFC45666C99E8AE922F7A807B7D07B62C995D79E2"
 }
 ```  
-**Response:**
+**Response since v21.0:**
 ```json
 {
-  "valid": "1",
-  "difficulty": "ffffffd21c3933f4",
-  "multiplier": "1.394647"
+  "valid_all": "1",
+  "valid_receive": "1",
```

Comment:
We might want to add details below about what the `valid_all` and `valid_receive` fields are indicating.

Reply:
Added above as we do with other RPCs. Also improved the callout specific to V21, which specifies a way to being able to use "valid_receive" even during the epoch ugprade. This method is valid (heh) right?

Feel free to use in the release notes as well.